pyro/contrib/epidemiology/compartmental.py

```diff
@@ -91,6 +93,10 @@ def __init__(self, compartments, duration, population):
         assert len(compartments) == len(set(compartments))
         self.compartments = compartments
 
+        if num_quant_bins not in [4, 8, 12, 16]:
```

Comment:
Can you please weaken this check and let the other file decide what values are allowed:
```py
assert isininstance(num_quant_bins, int)
assert num_quant_bins > 0
```

Reply:
done